DetectorDescription/DDCMS/interface/DDVolumeProcessor.h

```diff
@@ -0,0 +1,91 @@
+#ifndef DETECTOR_DESCRIPTION_DD_VOLUME_PROCESSOR_H
+#define DETECTOR_DESCRIPTION_DD_VOLUME_PROCESSOR_H
+
+#include "DD4hep/VolumeProcessor.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DetectorDescription/DDCMS/interface/DDSpecParRegistry.h"
+
+#include <string>
+#include <string_view>
+#include <vector>
+
+namespace cms {
+  
+  class DDVolumeProcessor : public dd4hep::PlacedVolumeProcessor {
+  public:
+
+    using Volume = dd4hep::Volume;
+    using PlacedVolume = dd4hep::PlacedVolume;
+    using PlacedVolumeProcessor = dd4hep::PlacedVolumeProcessor;
+    
+    DDVolumeProcessor() = default;
+    ~DDVolumeProcessor() override = default;
+
+    std::string_view stripNamespace(std::string_view v) {
```

Comment:
From our revised coding policy:
[4.3](https://github.com/cms-sw/cms-sw.github.io/blob/master/cms_coding_rules.md#4----technical-coding-rules) Header ﬁles must not contain any implementation except for class templates and code to be inlined.

This code should be moved to the cc file.